bindings/c/meshio.cpp

```diff
@@ -0,0 +1,72 @@
+#include "meshIO.h"
```

Comment:
Do you even want to make bindings for this? MeshIO is nothing special, just a testing convenience.

Reply:
It doesn't do a lot and I know there isn't much in that file, but I figured that generally people are going to want to write their models into some formats, and `assimp` bindings don't necessarily exist already. Should spare some duplication of effort if people don't have to write their own off-ramps. Outputting to ascii stl or 3mf is quick enough to do on your own, but the more efficient binary formats that assimp can provide are nice to have. Speaking of, I haven't read more into it, but the default that is output right now for stl is ascii, would be nice to have the option to write binstl from MeshIO. 

Is there a particular reason why you want to leave mesh output as a just for testing convenience rather than providing something as part of `Manifold`?

Reply:
Yes, the reason is that formats are complex and lots of code to maintain and other projects are already better at it than mine. I need some boundaries, or else a hell of a lot more collaborators! 

I refuse to support STL though: STL cannot represent a manifold mesh in any kind of reliable way, so using this library and outputting STL is self-defeating. Pretty much any other format will do, 3MF being probably ideal. 

Anyway, if you want to maintain it, you can, but be warned: someone's going to want UVs and then atlasing and eventually you'll have to write Blender all over.

Reply:
I didn't mean to suggest re-writing what assimp is providing and increase your responsibilities, just to do what is basically done now -- letting assimp do it. My decision to use MeshIO basically just that the interface with assimp is already being provided by it, so it is simple to provide it along with the C bindings. If assimp bindings (or some native/bound serialization library) exists in the language that the C bindings are being used from, then there is no problem, otherwise using Manifold will also mean coming up with those solutions on their own. Does that make sense?

Reply:
Maybe what I really mean is, `Manifold` can handle any kind of crazy vertex properties you want, but I'm afraid someone will just look at my crappy little MeshIO function that only outputs vertex colors and say, "oh I guess this library doesn't work on my meshes" and move on. There are just so many ways to deal with mesh data; I'm doing my best not to make any assumptions, but once you output to a file you've necessarily picked one.

Reply:
I don't think you have to worry about that too much! If people want more, they can always do it themselves with the data available from `Mesh`, I just think it's a nice batteries included solution to have output functions readily available, even if they aren't state of the art.